README.md

```diff
@@ -1,11 +1,14 @@
 ## Table of Contents
 - [Table of Contents](#table-of-contents)
 - [Overview](#overview)
+  - [Architecture](#architecture)
 - [Setup](#setup)
 - [State transitions and data composition](#state-transitions-and-data-composition)
```

Comment:
The following sections seem to be missing:

* [Epoch processing state transitions](https://github.com/PowerLoom/pooler/tree/feat/single_snapshotter#epoch-processing-state-transitions)
* [Internal Snapshotter APIs](https://github.com/PowerLoom/pooler/tree/feat/single_snapshotter#internal-snapshotter-apis)

Reply:
Added!